### DISCUSSION TITLE
Question 36: Autogenerated Mammals CE records have arbitrary Collection Event Codes assigned, why? [RB - INS]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#539] [Q&A] [answered]
Description: This came up while working out how to share sites/CEs data across host/specimen records. In consultation with Mammals, we noticed that Collection Events generated on the back end of Mammals sites records are unique to each catalog record via a CE code arbitrarily assigned.  

Sometimes we will have specimens linked to possibly multiple hosts since we know location and host species, but we cannot define which host specimen we should link to exactly. In these cases, Insects would end up generating our own CE record to avoid using a record with this arbitrarily assigned number since they’re specimen specific. Do the mammal CE’s need to remain unique to each specimen? What would be the best way to share these CE’s?

Example: Insects Catalog record: 3706144 has a host of Petaurista magnificus here at FMNH. (Possible FMNH Mammal catalog records are: 2604312-2604314), but we can’t define which.) If these all shared the same CE, we too could link to it. As they don’t, we created a close duplicate sans CE codes. (CE irn 3567695)

**Comment by rondlg** [comment, marked as answer]
Response: These can be collapsed into a single agreed record. JJ will work out where the auto-generated number is derived from.

**Reply by fmjjones** [reply]
https://github.com/fieldmuseum/EMu-Documentation/issues/540
